Assets/MixedRealityToolkit/_Core/Definitions/Utilities/OrientationType.cs

```diff
@@ -28,5 +28,22 @@ public enum OrientationType
         /// Zero rotation + 180 degrees
         /// </summary>
         FaceForwardReversed,
+        /// <summary>
+        /// Parent Relative Up
+        /// </summary>
+        FaceParentUp,
```

Comment:
We should take the htk->vNext port as an opportunity to rename FaceForward and FaceForwardReversed to FaceParentForward and FaceParentBack.

Reply:
bump

Reply:
Sure. Done.